controller/model/enrollment_mod_ca.go

```diff
@@ -181,8 +181,8 @@ func (module *EnrollModuleCa) Process(context EnrollmentContext) (*EnrollmentRes
 	return &EnrollmentResult{
 		Identity:      identity,
 		Authenticator: newAuthenticator,
-		Content:       []byte{},
-		ContentType:   "text/plain",
+		Content:       map[string]interface{}{},
+		TextContent:   []byte(""),
```

Comment:
I'm inferring that TextContent will implicitly set the Content-Type. if so - carry on - no response needed :) just pointing it out to make removing ContentType is intentional

Reply:
Text content can be many things, so yes it does have to set it.

This is a work-around for a larger issue that I didn't want to solve just now.

Reply:
I removed my previous comment. I misread your comment.

The removal was on purpose - it was the start to a larger refactor that I put off.

The HTTP `content-type` is determined by whatever `Producer` is used to write the content. A `Producer` is suggested by the `go-swagger` generated code, but can it may be one of many valid choices.

At this point - it isn't up to the enrollment process to determine the content type. `TextContext` is a hamfisted workaround as I didn't want to refactor the enrollment modules at this time.